src/main/java/frc/robot/Robot.java

```diff
@@ -4,9 +4,14 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
```

Comment:
Is joystick needed here?

Reply:
Looks like the constructor for Controls requires it